### PR TITLE
Fixes #30984 - translate host relative date creation

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js [hunk 1]
```diff
@@ -120,26 +121,20 @@ const HostDetails = ({
...
-                <RelativeDateTime
-                  date={response.updated_at}
-                  defaultValue="N/A"
-                />
-                )
-              </span>
-            ) : (
-              <Skeleton width={400} />
+          <SkeletonLoader status={status}>
+            {response.name && (
```
Comment (xprazak2) [inline webpack/assets/javascripts/react_app/components/HostDetails/index.js]: Why `response.name` if name is not used after `&&`?

**Comment by xprazak2** [comment]
Should we constrain the width of the skeleton bar?

loading-long

loading-long-icon

**Comment by xprazak2** [comment]
Looks good.

**Comment by xprazak2** [comment]
Nice, just getting a prop type failure:
Warning: Failed prop type: Invalid prop `children` supplied to `RelativeDateTime`, expected a ReactNode.

**Comment by amirfefer** [comment]
[foreman test]

**Comment by amirfefer** [comment]
[test foreman]

**Comment by amirfefer** [comment]
[test upgrade]

**Review by xprazak2 — APPROVED** [review]

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**